recipes/ofeli/5.x/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import copy, get, rmdir, replace_in_file, rename
+
+required_conan_version = ">=1.53.0"
+
+
+class OfeliConan(ConanFile):
+    name = "ofeli"
+    description = "An Object Finite Element Library"
+    license = "LGPL-3.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://ofeli.org/index.html"
+    topics = ("finite-element", "finite-element-library", "finite-element-analysis", "finite-element-solver")
+
+    package_type = "static-library"
```

Comment:
I did not find a reference offering only static library: https://github.com/rtouzani/ofeli/blob/master/src/CMakeLists.txt

Please, point where you found it.

Reply:
A shared build fails with:
```
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<unsigned long, 3ul>::~LocalVect()'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<unsigned long, 2ul>::~LocalVect()'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::Muscl::setReconstruction(OFELI::Vect<double> const&, OFELI::Vect<double>&, OFELI::Vect<double>&, unsigned long)'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `virtual thunk to OFELI::LaplaceDG2DP1::build()'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<double, 2ul>::get()'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<double, 3ul>::LocalVect()'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<unsigned long, 3ul>::LocalVect()'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<unsigned long, 2ul>::LocalVect()'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<double, 3ul>::operator()(unsigned long)'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<double, 2ul>::operator=(double const&)'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<double, 3ul>::get()'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<double, 3ul>::~LocalVect()'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LaplaceDG2DP1::build()'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<double, 1ul>::get()'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<double, 2ul>::operator()(unsigned long)'
ofelia3f8f09b1f9b5/p/lib/libofeli.so: undefined reference to `OFELI::LocalVect<double, 3ul>::operator=(double const&)'
```
on GCC.